com.unity.formats.alembic/Editor/Misc/UIHelper.cs

```diff
@@ -0,0 +1,69 @@
+using UnityEngine;
+using System;
+
+namespace UnityEditor.Formats.Alembic
+{
+    class UIHelper
+    {
+        const float k_IndentMargin = 15.0f;
+
+        static GUIStyle s_HelpBoxStyle = null;
+
+        static GUIStyle helpBoxStyle
+        {
+            get
+            {
+                if (s_HelpBoxStyle == null)
+                {
+                    s_HelpBoxStyle = new GUIStyle()
+                    {
+                        imagePosition = ImagePosition.ImageLeft,
+                        fontSize = 10,
+                        wordWrap = true,
+                        alignment = TextAnchor.MiddleLeft
+                    };
+                    s_HelpBoxStyle.normal.textColor = EditorStyles.helpBox.normal.textColor;
+                }
+                return s_HelpBoxStyle;
+            }
+        }
+
+        /// <summary>Draw a help box with a button.</summary>
+        /// <param name="message">The message, unlocalized.</param>
+        /// <param name="buttonLabel">The button text.</param>
+        /// <param name="action">When the user clicks the button, Unity performs this action.</param>
+        internal static void HelpBoxWithAction(string message, MessageType messageType, string buttonLabel, Action action)
```

Comment:
small detail : `MessageType messageType` doesn t have param name entry in the summary

Reply:
Thanks, here: [b0ba5c8](https://github.com/Unity-Technologies/com.unity.formats.alembic/pull/577/commits/b0ba5c84a88ce8ee305355ae11d375cd459a1ae5)